controllers/test_helpers.go

```diff
@@ -0,0 +1,45 @@
+package controllers
```

Comment:
In this way test related codebase looks like production code. I suggest to refactor the code to ensure test functions are only available in tests. If it is not possible a common solution to create a `controllertest` package for separation.

Reply:
Sorry, my bad, this package contains only tests :D